lib/network/api/resource/resource.go

```diff
@@ -0,0 +1,44 @@
+package resource
+
+import (
+	"github.com/nvellon/hal"
+)
+
+type APIResource interface {
```

Comment:
It just suggestion, `APIResource` -> `Resource`?

Reply:
Oh, Thanks! :-)